src/pages/agent/agent.tsx

```diff
@@ -88,6 +88,7 @@ export const Agent = () => {
             </TableActionsProvider>
           )}
         />
+        <Redirect push to="/" />
```

Comment:
Why it need`s , and use getAdminPath for these cases

Reply:
this logic works after error in t2c path: .../test2code/WRONG